helm_deploy/values-test.yaml

```
@@ -131,6 +131,7 @@ generic-service:
     ncc-tsc-team-9: "195.95.131.0/24"
     ncc-mvss-team-1: "5.148.8.192/26"
     dxw-vpn: "54.76.254.148/32"
+    cyberlab: "51.38.77.66"
```

add this in CIDR format.
```suggestion
    cyberlab: "51.38.77.66/32"
```